drone.yml

```diff
@@ -0,0 +1,46 @@
+pipeline:
```

Comment:
needs more more constrains for when to build the pipelines. SEE latest version of /brp_app